core/spec/models/spree/shipment_spec.rb

```diff
@@ -750,4 +750,18 @@
       end
     end
   end
+
+  describe ".by_store" do
+    it "returns shipments by store" do
+      olivanders_store = build(:store, name: 'Olivanders')
```

Comment:
More a preference. They get created eventually. I don't feel strongly one way or the other. Will change!